src/biotite/structure/graphics/rna.py

```diff
@@ -0,0 +1,260 @@
+# This source code is part of the Biotite package and is distributed
+# under the 3-Clause BSD License. Please see 'LICENSE.rst' for further
+# information.
+
+__name__ = "biotite.structure.graphics"
+__author__ = "Tom David Müller"
+__all__ = ["plot_nucleotide_secondary_structure"]
+
+import shutil
+import numpy as np
+import matplotlib.pyplot as plt
+from .. import pseudoknots
+from ...application.viennarna import RNAplotApp
+
+
+def plot_nucleotide_secondary_structure(
```

Comment:
All plotting functions in Biotite use an `Axes` as first parameter to give the user the flexibility to embed the visualization in a multi-plot figure. You could have a look at https://www.biotite-python.org/_modules/biotite/sequence/graphics/alignment.html#plot_alignment as an example.